rulebook.tex

```diff
@@ -1295,12 +1303,6 @@ \subsection{Interruptions and Robot Maintenance}
 so rarely as not to interfere with the overall game flow (also
 cf.~\refsec{sec:during-match}).
 
-If a robot needs to be taken out for the second time, either on
```

Comment:
Thanks for pointing that out. I somehow accidentally deleted that paragraph. Should be re-inserted now. It also states that the robot is now disqualified if it needs to be taken out for the third time, in order to account for the now allowed second maintenance.